store/exercise.go

```diff
@@ -283,11 +283,32 @@ type exercisestore struct {
 	hooks     []func([]Exercise) error
 }
 
+func (es *exercisestore) UpdateExercisesFile(path string) (ExerciseStore, error) {
+	var conf struct {
+		Exercises []Exercise `yaml:"exercises"`
+	}
+	f, err := ioutil.ReadFile(path)
+	if err != nil {
+		return nil, err
+	}
+	err = yaml.Unmarshal(f, &conf)
```

Comment:
Isn't this already implemented in https://github.com/aau-network-security/haaukins/blob/537393e9e1c1ee4f5a7efe5350b6cdd9b18e9ca9/store/exercise.go#L411.

I think it makes a lot more sense to have it here, maybe it can call into this function or something.

Reply:
changed, thanks.